plugins/modules/identity/ipa/ipa_dnszone.py

```diff
@@ -100,6 +115,17 @@ def dnszone_add(self, zone_name=None, details=None):
             item=itens
         )
 
+    def dnszone_mod(self, zone_name=None, details=None):
+        itens = {}
```

Comment:
Probably this should be:
```suggestion
        items = {}
```
(Same in the next lines.)